src/System.Data.Odbc/src/Common/System/Data/Common/UnsafeNativeMethods.cs

```diff
@@ -18,24 +18,23 @@ internal static partial class UnsafeNativeMethods
         //
         // ODBC32
         //
-        [DllImport(ExternDll.Odbc32)]
+        [DllImport(Interop.Libraries.Odbc32)]
```

Comment:
These will be moved to the normal interop structure in a subsequent PR, at which point you'll flip Libraries back to being private instead of internal?

Reply:
the access identidifier on Interop.Libraries has been inconsistent between different platforms as you see in the diff list.

This class is not the only place I'm using Interop.Libraries.Odbc32. I'm also using this field in 
src/System.Data.Odbc/tests/Helpers.cs

I won't be able to access Odbc32 if I set access modifier to private.

Reply:
Ok. But with regards to this file, it'll be moved/changed in a someone PR?

Reply:
I'm not aware if it's going to be moved or changed.

Reply:
@stephentoub moved pinvokes to interop